easybuild/easyconfigs/a/Amber/Amber-16-intel-2017b-AmberTools-17-patchlevel-0-7.eb

```diff
@@ -0,0 +1,50 @@
+name = 'Amber'
+version = '16'
+ambertools_ver = '17'
+# Patch levels from http://ambermd.org/bugfixes16.html and http://ambermd.org/bugfixesat.html
+patchlevels = (0, 7)  # (AmberTools, Amber)
+versionsuffix = '-AmberTools-%s-patchlevel-%s-%s' % (ambertools_ver, patchlevels[0], patchlevels[1])
+
+homepage = 'http://ambermd.org/amber.html'
+description = """Amber (originally Assisted Model Building with Energy
+ Refinement) is software for performing molecular dynamics and structure
+ prediction."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+sources = [
+    'Amber%(version)s.tar.bz2',
+    'AmberTools%s.tar.bz2' % ambertools_ver,
+]
+
+checksums = [
+    '3b7ef281fd3c46282a51b6a6deed9ed174a1f6d468002649d84bfc8a2577ae5d',
+    '4fbb2cf57d27422949909cc6f7e434c9855333366196a1d539264617c8bc5dec',
+]
+
+patches = [
+    'Amber-%%(version)s-AT-%s_fix-hardcoding.patch' % ambertools_ver,
+    'Amber-%%(version)s-AT-%s-fix_intel_mpi_compiler_version_detection.patch' % ambertools_ver,
+    'Amber-%%(version)s-AT-%s-fix_mkl_include_path.patch' % ambertools_ver,
+    'Amber-%%(version)s-AT-%s-use_fftw_from_mkl_or_external.patch' % ambertools_ver,
+    'Amber-%%(version)s-AT-%s-use_FFTW_FFT_instead_of_PUBFFT.patch' % ambertools_ver,
+    'Amber-%%(version)s-AT-%s-fix_sander_link_with_external_fftw.patch' % ambertools_ver,
+    'Amber-%%(version)s-AT-%s-fix-cpptraj-dont-use-static.patch' % ambertools_ver,
+    'Amber-%%(version)s-AT-%s-fix_incorrect_omp_directive_rism.patch' % ambertools_ver,
+    'Amber-%%(version)s-AT-%s-fix_missing_openmp_at_link.patch' % ambertools_ver,
+    'Amber-%%(version)s-AT-%s-use_CUSTOMBUILDFLAGS_for_nab.patch' % ambertools_ver,
+    'Amber-%(version)s-fix_missing_build_target.patch',
+]
+
+dependencies = [
+    ('netCDF', '4.4.1.1'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('Python', '2.7.14'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+]
+
+runtest = False
```

Comment:
@akesandgren The existing `Amber` easyconfigs don't have this, why is this needed?

Reply:
Needed and needed.
Th extra dependencies are there because Amber searches for them and can use them.

The patches I haven't had time to push upstream yet but are important for a correct build.

Reply:
@akesandgren I was referring to `runtest = False` :)

Reply:
Ahhh, can't remember at the moment, will have to check...

Reply:
Found some bugs still remains to be solved. This easyconfig is not complete, there is at least one patch file missing.
Plus some failures during tests.

Will try to figure out the test failures.